docs/pages/cli-docs.mdx

```diff
@@ -532,22 +532,36 @@ registered devices using [`tsh mfa ls`](cli-docs.mdx#tsh-mfa-ls).
 
 ## tctl
 
-`tctl` is an admin CLI tool used to administer a Teleport cluster. It connects
-to an Auth Server, meaning that it must be run on the same host with the role
-`auth` . If you run `tctl` on a non- `auth` node it will show an error.
+`tctl` is an admin CLI tool used to administer a Teleport cluster. 
 
 `tctl` allows a cluster administrator to manage all resources in a cluster
 including nodes, users, tokens, and certificates.
 
 `tctl` can also be used to modify the dynamic configuration of the cluster, like
 creating new user roles or connecting trusted clusters.
 
+By default, `tctl` connects to a local Auth server, meaning it will fail if you 
+attempt to run `tctl` on a non-`auth` node.
+
+However it can also connect to a remote Auth server if an identity file and Auth/Proxy
+server IP address are provided. 
+
+<Admonition type="note">
+  Note that when a `tctl` command is run locally on an Auth server, the audit logs will 
+  show that it was performed by the Auth server itself. To properly audit admin actions
+  at scale, it is important to close off direct access to the Auth server and ensure
```

Comment:
By "close off direct access" do you mean at a network admin level (e.g. disabling direct ssh access to the box), or something we can control in Teleport itself? Is there something we can link to to provide guidance, or is this something that will vary too much from install to install to be useful?

Reply:
I meant ssh access via teleport, I'll reword this better. I think the ways this can be done varies so I'd rather not recommend something that doesn't fit all use cases.